charts/kube-otel-stack/templates/collector.yaml

```diff
@@ -182,6 +182,14 @@ rules:
   verbs: ["get", "list", "watch"]
 - nonResourceURLs: ["/metrics", "/metrics/cadvisor"]
   verbs: ["get"]
+{{- if $collector.kubeletEvents }}
```

Comment:
im wondering if we should just make it possible to add RBAC groups in here i.e. you would do something like 
```yaml
{{- with $collector.extraRbac }}
{{ . }}
{{- end }}
```

or something like that. That way you can add as many rbac requests as you want.